actors/market/src/ext.rs

```diff
@@ -150,6 +149,8 @@ pub mod reward {
 }
 
 pub mod power {
+    use fil_actors_runtime::reward::FilterEstimate;
```

Comment:
out of interest - why are you putting it in here in this case?

Reply:
It's used here and automatic refactoring tools moved it. I'll move it back.

Reply:
Fixed.